ui/lib/core/addon/components/toolbar-actions.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * @module ToolbarActions
+ * `ToolbarActions` components are containers for Toolbar actions.
+ *
+ * @example
+ * ```js
+ * <ToolbarActions>Toolbar Actions</ToolbarActions>
```

Comment:
Same comment as above -- it'd be helpful to show that the children of `ToolbarActions` are generally `ToolbarLinks`

Reply:
I agree, but I couldn't decide if that would be more confusing. Done!